src/db/sqlite/lrsql/sqlite/record.clj

```diff
@@ -104,7 +104,9 @@
       (migrate-timestamps-activity-profile-02! tx)
       (migrate-timestamps-activity-profile-03! tx)
       (migrate-timestamps-activity-profile-04! tx))
-    (update-schema-simple! tx alter-credential-to-scope-scope-datatype-v2!)
+    (when-not (= 1 (:scope_enum_updated
```

Comment:
Don't you mean to add this comment to the Postgres record code?

Reply:
But I suppose. It was simply easier to return the whole query in the PostgreSQL code, and since we don't actually care about the contents (just whether it's null or not null) I figured we could get away with that.

Reply:
I mean it in both. The hug fn should directly return T/F/nil as previous precedent.

Reply:
I suppose an argument can be made that _those_ queries should also be changed to return simpler values.

Reply:
yeah they should be. but in this case im just asking you to simplify the new one

Reply:
https://github.com/yetanalytics/lrsql/pull/407/commits/528779f69eff900ba8e72ea2f7b3700959ab398d and https://github.com/yetanalytics/lrsql/pull/407/commits/59a8e10df31eb7319ea4de31f25e0adb76f86a2c